ironmon_tracker/Tracker.lua

```diff
@@ -341,8 +341,8 @@ end
 -- If the Pokemon is being tracked, return its note; otherwise default note value = ""
 function Tracker.getNote(pokemonID)
 	local trackedPokemon = Tracker.getOrCreateTrackedPokemon(pokemonID)
-	if trackedPokemon.note == nil then
-		return ""
+	if trackedPokemon == nil or trackedPokemon.note == nil then
```

Comment:
Up to you if you want to revert this, but the `Tracker.getOrCreateTrackedPokemon(pokemonID)` will never return `nil` by design, since it either gets the current Pokemon or creates a new one if one doesn't exist; it always returns something that can be tracked.